### PR TITLE
Deprecate and remove premium_price attribute from registrar order responses

### PR DESCRIPTION
**Pull request by weppos**
This is inconsistent with most of our code, and if we will ever return a price it will be returned for every order.

## Clients

- [x] dnsimple/dnsimple-ruby#163
- [x] dnsimple/dnsimple-go#67
- [x] dnsimple/dnsimple-elixir#120
- ~dnsimple/dnsimple-node~ (not required)
- [x] dnsimple/dnsimple-java#10

### CODE REVIEW

**Comment by weppos** [comment]
Going to update the clients once this PR is approved.

File: content/v2/openapi.yml [hunk 1]
```diff
@@ -4112,7 +4100,7 @@ components:
             properties:
               period:
                 type: integer
-              preimum_price:
+              premium_price:
```
Comment (aeden) [inline content/v2/openapi.yml]: Isn't premium_price supposed to be removed?

Comment (weppos) [reply content/v2/openapi.yml]: Nope, this is the parameter. Not the response field.

Comment (aeden) [reply content/v2/openapi.yml]: Ah, ok. 👍

**Pull request merged**